android/assets/jsons/translations/German.properties

```diff
@@ -990,8 +990,7 @@ Current mods = Installierte Modifikationen
 Downloadable mods = Verfügbare Modifikationen
 Next page = Nächste Seite
 Open Github page = Auf Github öffnen
-Enable as permanent visual mod = Als permanente visuelle Modifikation aktivieren
-Disable as permanent visual mod = Als permanente visuelle Modifikation deaktivieren
+Permanent visual mod = Permanente audiovisuelle Mod
```

Comment:
In template.properties heißt der String "Permanent **audio**visual mod"

Reply:
Aua, danke. Und ich habe es nur abgekürzt weil da wenig Platz ist - ist in Portrait eh eine Katastrophe und bräuchte eine gute Idee...